src/plugins/discover/public/application/components/discover_legacy.tsx

```diff
@@ -203,117 +192,143 @@ export function DiscoverLegacy({
+                          searchTitle={opts.savedSearch.lastSavedTitle}
+                          onAddColumn={addColumn}
+                          onFilter={onAddFilter}
+                          onMoveColumn={onMoveColumn}
+                          onRemoveColumn={onRemoveColumn}
+                          onSort={onSort}
                         />
-                      </h2>
-                      {rows && rows.length && (
-                        <div className="dscDiscover">
-                          <DocTableLegacy
-                            columns={state.columns || []}
-                            indexPattern={indexPattern}
-                            minimumVisibleRows={minimumVisibleRows}
-                            rows={rows}
-                            sort={state.sort || []}
-                            searchDescription={opts.savedSearch.description}
-                            searchTitle={opts.savedSearch.lastSavedTitle}
-                            onAddColumn={addColumn}
-                            onFilter={onAddFilter}
-                            onMoveColumn={onMoveColumn}
-                            onRemoveColumn={onRemoveColumn}
-                            onSort={onSort}
-                          />
-                          <a tabIndex={0} id="discoverBottomMarker">
-                            &#8203;
-                          </a>
-                          {rows.length === opts.sampleSize && (
-                            <div
-                              className="dscTable__footer"
-                              data-test-subj="discoverDocTableFooter"
+                        <a tabIndex={0} id="discoverBottomMarker">
+                          &#8203;
+                        </a>
+                        {rows.length === opts.sampleSize && (
+                          <div className="dscTable__footer" data-test-subj="discoverDocTableFooter">
+                            <FormattedMessage
+                              id="discover.howToSeeOtherMatchingDocumentsDescription"
+                              defaultMessage="These are the first {sampleSize} documents matching
+                  your search, refine your search to see others."
+                              values={{ sampleSize: opts.sampleSize }}
+                            />
+
+                            <EuiButtonEmpty
+                              onClick={() => {
+                                // depending on screen size there are different
+                                if (!isMobile() && scrollableDesktop.current) {
+                                  scrollableDesktop.current.scrollTo(0, 0);
+                                }
+                                if (isMobile() && scrollableMobile.current) {
+                                  scrollableMobile.current.scrollTo(0, 0);
+                                }
+                              }}
```

Comment:
Let's also move focus to something at the top of the page so that keyboard users get the same benefit.

Reply:
Like focus to the "scroll to Bottom" link? don't know which element would make sense?

<img width="732" alt="Bildschirmfoto 2020-11-20 um 06 09 26" src="https://user-images.githubusercontent.com/463851/99762197-3dadfa00-2af8-11eb-96e6-780b9bfcca76.png">

Reply:
Maybe the `main` we setup above? (You'll need to add `tabIndex={-1}` to the `main` to move focus to it.)

Reply:
thx, i've changed it to target `main`